### DISCUSSION TITLE
uc_gui_click_x_y in parallel

### DISCUSSION THREAD

**Discussion by ApoNikol** [seleniumbase/SeleniumBase#3072] [Q&A] [answered]
Good morning,
I am using the new function uc_gui_click_x_y(x,y,timeframe).
The pyautogui function works perfectly with one google tab opened, but when I add more of them in parallel it only works at the one in the front.
I believe that is because of the mouse click working only in the tab that is currently in the front of the screen.
Is there a way for it to work with this function, or if not is there a way to click by coordinates with invisible mouse so I can avoid the problem?
Thank you!

**Comment by mdmintz** [comment, marked as answer]
`PyAutoGUI` actions cannot be run in parallel.
There's only one mouse pointer on the screen, and it can only click on one location at a time.
This applies to the **UC Mode** actions that use `PyAutoGUI`.

**Reply by ApoNikol** [reply]
Thanks a lot for the reply Michael! I will consider another solution.
